connector/controllers/main.py

```diff
@@ -112,11 +112,10 @@ def retry_postpone(job, message, seconds=None):
             retry_postpone(job, unicode(err), seconds=err.seconds)
             _logger.debug('%s postponed', job)
 
-        except (FailedJobError, Exception):
+        except:
```

Comment:
@sbidoul that's what I thought too. But the same applies to `FailedJobError`, and this one is mentioned explicitly here. `git blame` says you wrote this line, do you remember why you added `FailedJobError`? And do you agree in general to catch unconditionally here? I'd think we want to persist the failed state no matter what happened, right?